SpaceDuck.UserService/SpaceDuck.Common/Models/RoomConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace SpaceDuck.Common.Models
+{
+    public class RoomConfiguration
+    {
+        public string PlayerOwnerId { get; set; }
```

Comment:
Tutaj jeszcze powinna być ilość tur, ustawiamy ją przy tworzeniu gry.
Plus, jakaś lista osób, które zadeklarowały chęć rysowania, nie każdy musi.

Reply:
I gdzie trzymamy punkty konkretnego pokoju?
Nie powinny być tutaj?
Bo każdy pokój też powinien mieć swój ranking na koniec gry.

Reply:
punkty konkretnego pokoju ? co to na co ? Ranking myślę że może być trzymany przy np liście graczy można zmienić na mapę <playerId, points>

Reply:
z tą chęcią do tysowania jeszcze nie myślałem ale dodam jakoś

Reply:
Okej, możemy trzymać <>.